src/lib/Scenes/Home/Components/Sales/Components/ZeroState/__tests__/index-tests.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react"
+import "react-native"
+import * as renderer from "react-test-renderer"
+import { ZeroState } from "../index"
+
+jest.mock("WebView", () => "WebView")
+jest.mock("../index.html", () => "")
+
+it("looks correct when rendered", () => {
+  const auctions = renderer.create(<ZeroState />)
+  expect(auctions).toMatchSnapshot()
```

Comment:
FWIW, this test is really telling us nothing with the snapshot ^

Maybe add tests around `onShouldStartLoadWithRequest` which has some logic?